src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2239,14 +2239,14 @@ bool entitiesQuery
     catch (const std::exception &e)
     {
       *err = e.what();
-      LM_E(("Runtime Error (exception in nextSafe(): %s)", e.what()));
+      LM_E(("Runtime Error (exception in nextSafe() at %s:%d: <%s> - query: <%s>)", __FUNCTION__, __LINE__, e.what(), query.toString().c_str()));
       releaseMongoConnection(connection);
       return false;
     }
     catch (...)
     {
       *err = "generic exception at nextSafe()";
-      LM_E(("Runtime Error (generic exception in nextSafe())"));
+      LM_E(("Runtime Error (generic exception in nextSafe() at %s:%d - query: <%s>)", __FUNCTION__, __LINE__, query.toString().c_str()));
```

Comment:
LM_E suplies the line and function of the point of the code in which LM_E appears, i.e. within get*Field(). But I need to know the function and line corresponding to the caller function.